recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gensim" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "7d0dccc7d2c576e270037949874800b7cfbc86ef081ff981483f612cd18e223f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.11.3
+    - scipy >=0.18.1
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+  run:
+    - python
+    - numpy >=1.11.3
+    - scipy >=0.18.1
+    - six >=1.5.0
+    - smart_open >=1.2.1
+    
+test:
+  requires:
+    - morfessor
+    - scikit-learn
+    
+  imports:
+    - gensim
+    - gensim.corpora
+    - gensim.models
+    - gensim.models.wrappers
+    - gensim.parsing
+    - gensim.scripts
+    - gensim.similarities
+    - gensim.sklearn_integration
+    - gensim.summarization
+    - gensim.test
+    - gensim.topic_coherence
+    
+  commands:
```

Comment:
This is a problem with `sklearn==0.19.0`, we already fix it in `develop` branch. Now you can "hardcode" sklearn to `0.18.2` to avoid this (and remove in next gensim release)